### ISSUE TITLE
"Fields must be qualified with table name" rule is not detecting unqualified references

### ISSUE THREAD

**Issue by akompanas** [Riverside-Software/sonar-openedge#1228]
```
DEFINE TEMP-TABLE tbl NO-UNDO
    FIELD int_value AS INT.

CREATE tbl.
tbl.int_value = 123.

DO i = 1 to 5:
END.

DISPLAY int_value.
```
The rule "Fields must be qualified with table name" (Rule ID: rssw-oe:compiler.warning.14789) does not flag the unqualified use of `int_value` neither at DO, nor at DISPLAY lines, in this case leading to a bug in DO inadvertently resolving to the temp-table field.

**Comment by gquerret** [comment]
Works for me:
<img width="1190" height="718" alt="Image" src="https://github.com/user-attachments/assets/ec5e6b24-be5e-4fca-a7c1-ac63b4073907" />

You must compile with PCT to have the compiler warnings imported in SonarQube. If you don't use PCT, you must have the expected directory structure to import the compiler warnings (`.pct` directory with `.warnings` files)

**Comment by akompanas** [comment]
Thanks for clarification. I can't seem to find those special files. Is there a special flag that controls their location / creation?

**Comment by gquerret** [comment]
Clone this [sample project](https://github.com/Riverside-Software/sonaroe-test-project) and execute `ant build`. The files will be generated in `target/build/.pct`. You will find the details in the Ant `build.xml` file.

**Comment by akompanas** [comment]
Just to add more context, builds use Ant + PCT and they were set up according to this instruction:
https://github.com/Riverside-Software/sonar-openedge/wiki/Getting-started

However, I don't see neither .pct, nor .warning anywhere in the build directory and across hundreds of projects this rule has zero hits, which I find hard to believe.

**Comment by akompanas** [comment]
Is it possible that adding other options, like specifying xref and/or preprocess dir stops this from working?

**Comment by gquerret** [comment]
I don't know your context, but PCT always creates a `.pct` subdirectory in the build directory. Setting xrefDir will change this directory name. The list of attributes can be found [here](<https://wiki.rssw.eu/pct/PCTCompile>).

**Comment by akompanas** [comment]
OK, so there are two things:
1. You need to specify require[COMPILATION_OPTION_HERE] flags for PCTCompile - otherwise the warnings are not generated
2. The .warnings file goes to xrefDir.

Could you please update the setup instruction here?
https://github.com/Riverside-Software/sonar-openedge/wiki/Getting-started

**Comment by gquerret** [comment]
Added `requireFullKeywords="true"` to have these warnings visible in SonarQube.

**Comment by akompanas** [comment]
The option is `requireFieldQualifiers` for this particular case. But maybe with full keywords it's more annoying :)
